scripts/ci/jobs/jobs.yml

```diff
@@ -270,6 +270,23 @@
       - skydive-test:
           test: scripts/ci/run-compile-tests.sh
 
+- job:
+    name: skydive-vpp-tests
```

Comment:
Now that we have a skydive-hw-tests job, I think the VPP tests can be moved to it